jhbuildrc-custom

```diff
@@ -9,7 +9,7 @@ else:
 
 checkoutroot = os.path.join(os.path.dirname(prefix), 'src')
 buildroot = os.path.join(os.path.dirname(prefix), 'build')
-print("Building in %s", buildroot)
+print("Building in", buildroot)
```

Comment:
This is Python, so the problem is really the comma not the placeholder. The line should be
```
 print("Building in %s" % buildroot)
```

However, the print was just a diagnostic I used while testing the adjustments I made yesterday and forgot to remove before committing.